python/Ganga/Core/GangaRepository/SessionLock.py

```diff
@@ -959,3 +841,4 @@ def session_to_info(self, session):
         except Exception, err:
             logger.debug( "Session Info Exception: %s" % err)
             return session
+
```

Comment:
What's W292? PEP 8 says nothing about blank lines at the end of files. However, there is a general principle when working with text files in version control that they should end with a `\n`. This file already seemed to end with one and you've added a second one.